src/dnx.common/tpa.cpp

```diff
@@ -18,7 +18,6 @@ const std::vector<const dnx::char_t*> CreateTpaBase(bool native_images)
             _X("Microsoft.Dnx.Host.CoreClr.ni.dll"),
             _X("Microsoft.Dnx.Loader.ni.dll"),
             _X("Microsoft.Dnx.Runtime.Abstractions.ni.dll"),
-            _X("mscorlib.ni.dll"),
```

Comment:
From our testing it looks like mscorlib is already treated as a TPA, but perhaps that's wrong. It certainly works fine without it in the TPA.